tests/foreman/ui/test_subnet.py

```diff
@@ -81,19 +63,30 @@ def test_positive_end_to_end(session):
             'subnet.network_prefix': '24',
             'subnet.ipam': 'EUI-64',
             'subnet.mtu': '1600',
+            'domains.resources.assigned': [module_dom.name],
         })
-        assert session.subnet.search(name)[0]['Name'] == name
-        subnet_values = session.subnet.read(name)
-        assert subnet_values['subnet']['name'] == name
-        assert subnet_values['subnet']['description'] == description
-        assert subnet_values['subnet']['protocol'] == 'IPv6'
-        assert subnet_values['subnet']['network_address'] == network_address
-        assert subnet_values['subnet']['network_prefix'] == '24'
-        assert subnet_values['subnet']['ipam'] == 'EUI-64'
-        assert subnet_values['subnet']['mtu'] == '1600'
+        sn = entities.Subnet().search(query={'search': 'name={0}'.format(name)})
+        assert len(sn) > 0, 'Subnet {0} expected to exist, but it\'s not listed'
```

Comment:
If you have single-quote character inside string, please surround string with double-quote characters. Allows you to drop backslash.
Also, seems you are missing `.format()` method call on assertion failure message.

Reply:
I'm following an ancient codestyle rule, where it was settled on using single quotes everywhere. I don't mind escaping a character here and there - is that so big issue?

Reply:
Will fix the missing `format()`

Reply:
That rule is justified as attempt to reduce visual noise. Backslash inside a string is a definition of visual noise, if you ask me.

But no, this is not big, blocking issue. More in would-be-nice-to-have and personal preference territory.